### PR TITLE
Add Generator object

### PR DESCRIPTION
**Pull request by LtRipley36706**
Included test data for training dungeon part 1 which should provide a few objects to help with system build out, such as inventory, respawning, object tracking between clients...

### CODE REVIEW

File: Source/ACE/Entity/Generator.cs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+using System.Diagnostics;
+
+namespace ACE.Entity
+{
+    public class Generator : WorldObject
+    {
+        public Generator(ObjectGuid guid, AceObject baseAceObject)
+            : base((ObjectType)baseAceObject.ItemType, guid)
+        {
+            Name = baseAceObject.Name ?? "NULL";
```
Comment (Mogwai-TheFurry) [inline Source/ACE/Entity/Generator.cs]: why all the commented code in a new class?  Can you document why all this isn't necessary and just nuke it?

File: Source/ACE/Entity/Monster.cs [hunk 2]
```diff
@@ -23,6 +25,74 @@ public Monster(AceObject aceO)
             else
                 this.WeenieClassid = (ushort)(aceO.WeenieClassId - 0x8000);
```
Comment (Mogwai-TheFurry) [inline Source/ACE/Entity/Monster.cs]: Why are we manually copying all this data in a derived type?  Seems like this is a symptom of a bad constructor chain.

Comment (LtRipley36706) [inline Source/ACE/Entity/Monster.cs]: TBH, I just noticed that there was an issue with monsters showing up on radar, so I c&p'd from DebugObject which corrected that. Didn't go beyond that. I don't get what the weenieclassid ushort stuff is either.

File: Source/ACE/Entity/Generator.cs (hunk 1 above)
Comment (LtRipley36706) [inline Source/ACE/Entity/Generator.cs]: Started from DebugObject and then hard coded propertied I'm assuming to be immutable between generators and likely could be left out of the database in the future entirely. Left it commented for easier undo, can remove in the future when certain of the path. I'm also making the assumption that code from GeneratorFactory eventually ends up in this class instead and we make use of actionchaining to spawn/despawn, etc

**Pull request merged**